cpp/modmesh/buffer/pymod/wrap_SimpleArray.cpp

```diff
@@ -84,14 +84,20 @@ class MODMESH_PYTHON_WRAPPER_VISIBILITY WrapSimpleArray
                             stride.push_back(arr_in.strides(i) / itemsize);
                         }
 
-                        const bool is_c_contiguous = (arr_in.flags() & py::array::c_style) == py::array::c_style;
-                        const bool is_f_contiguous = (arr_in.flags() & py::array::f_style) == py::array::f_style;
+                        auto && is_c_contiguous = [&]() -> bool
+                        {
+                            return (arr_in.flags() & py::array::c_style) == py::array::c_style;
+                        };
+                        auto && is_f_contiguous = [&]() -> bool
+                        {
+                            return (arr_in.flags() & py::array::f_style) == py::array::f_style;
+                        };
```

Comment:
I'm not sure whether wrapping with lambda function is better than initializing the boolean variable.

Reply:
These lambda implementations work the same as direct evaluation.  They do not simplify anything either.  Please revert them.